doc/api/worker_threads.md

```diff
@@ -1127,6 +1127,17 @@ Calling `unref()` on a worker allows the thread to exit if this is the only
 active handle in the event system. If the worker is already `unref()`ed calling
 `unref()` again has no effect.
 
+## Notes
+
+### Launching worker threads from preload scripts
+
+Take care when launching worker threads from preload scripts (scripts loaded
+and run using the `-r` command line flag). Worker threads automatically inherit
+the command line flags from the running process and will preload the same
+preload scripts as the main thread. If the preload script unconditionally
```

Comment:
This should specify that this only happens *by default*, when `execArgv` is not passed to the Worker constructor